java/src/jmri/jmrix/openlcb/OlcbThrottle.java

```diff
@@ -43,26 +54,43 @@ public OlcbThrottle(DccLocoAddress address, SystemConnectionMemo memo) {
         if (iface.getDatagramService() == null) {
             log.error("Failed to access Datagram Service");
         }
+        ot = new TractionThrottle(iface);
+        NodeID nid;
         if (address instanceof OpenLcbLocoAddress) {
-            oti = new ThrottleImplementation(
-                    ((OpenLcbLocoAddress) address).getNode(),
-                    iface.getNodeStore(),
-                    iface.getDatagramService()
-            );
+            nid = ((OpenLcbLocoAddress) address).getNode();
         } else {
-            oti = new ThrottleImplementation(
-                    this.address.getNumber(),
-                    this.address.isLongAddress(),
-                    iface.getNodeStore(),
-                    iface.getDatagramService()
-            );
+            int dccAddress = this.address.getNumber();
+            if (this.address.isLongAddress()) {
+                nid = new NodeID(new byte[]{6, 1, 0, 0, (byte) (((dccAddress >> 8) & 0xFF) | 0xC0),
+                        (byte) (dccAddress & 0xFF)});
```

Comment:
It would be very good if these numbers were documented.
```
// 6 = Something
// 1 = Something else
// 0 = ???
// 0 = ???
// 0 = ???
// 0xFF = Keep only the lowest 8 bits
// 0xC0 = ??
nid = new NodeID(new byte[]{6, 1, 0, 0, (byte) (((dccAddress >> 8) & 0xFF) | 0xC0),
        (byte) (dccAddress & 0xFF)});
```

Reply:
Thank you for the review.

I added a comment documenting how these numbers came about.
Also added release notes entries for the changes.